Nodejs/Setup/Custom_Action/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Custom_Action")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Custom_Action")]
+[assembly: AssemblyCopyright("Copyright ©  2015")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]

```

Comment:
Reference ProjectBefore.settings and ProjectAfter.settings rather than redefining a bunch of properties that are already in AssemblyInfoCommon. https://github.com/Microsoft/nodejstools/blob/master/Common/Tests/AssemblyInfoCommon.cs